packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -53,7 +53,6 @@ export default class Atomic extends Command {
   )
   public async run() {
     await this.createProject();
-    this.displayFeedbackAfterSuccess();
```

Comment:
handled in plop directly, which is better since it will also display the message standalone